wallet/src/de/schildbach/wallet/ui/invite/InviteHandler.kt

```diff
@@ -108,7 +125,7 @@ class InviteHandler(val activity: AppCompatActivity, private val analytics: Anal
                         }
                         walletApplication.wallet != null -> {
                             log.info("the invite is valid, starting AcceptInviteActivity with invite: ${invite.link}")
-                            mainTask.startActivity(activity.applicationContext, AcceptInviteActivity.createIntent(activity, invite, false), null)
+                            mainTask.startActivity(activity.applicationContext, MainActivity.createIntent(activity, invite), null)
```

Comment:
If there is a wallet created, then we will activate the Home Screen (MainActivity) with the invite for processing after the user has entered their PIN (if the app is logged out)

Reply:
`mainTask.startActivity` is being used instead of `activity.startActivity` to get around some duplicate instance problems that happen as a result of setting `taskAffinity=""` from previous work.

However, in this story, I am also adding `taskAffinity=""` to the application level, so all activities will have the same affinity.  This may help reduce or eliminate some duplicate instance problems and render other fixes we had to being unnecessary.  There is another story to address duplicate instances.

Reply:
Maybe better to change the message in log.info as well